opensearch/opensearch-kafka/src/config.py

```diff
@@ -0,0 +1,11 @@
+KAFKA_URI = "tk-kafka-demo-tkuppinen-demo.aivencloud.com:26170"
```

Comment:
instruction in readme to update broker URI?
[this](https://github.com/aiven/aiven-examples/blob/9ea5a8514aca4f9e42fd48a16426b93da47351cf/kafka/kafka-streams/makeconf.sh#L4) can be helpful to grab the URI automatically

Reply:
done